src/app/(food)/details.tsx

```diff
@@ -274,7 +287,7 @@ export default function FoodDetail(): JSX.Element {
                     <Text
                         style={[styles.notesText, { color: colors.labelColor }]}
                     >
-                        {t('details.footer')}
+                        {t(isGerman() ? 'details.footerOrg' : 'details.footer')}
```

Comment:
Additionally, this seems weird. I would suggest something like this:

```ts
`${!isGerman() ? t('details.translated') : ''} ${t('details.footer')}`
```

just for the simplicity in the lang files.

Currently the disclaimer texts are always translated two times even though only one is always used.

Otherwise we could just add the disclaimer to the footer translation itself. due to the fact that we only have german sources and other langs are always translated.